AutomatedTesting/Gem/PythonTests/Terrain/EditorScripts/Terrain_World_ConfigurationWorks.py

```diff
@@ -0,0 +1,163 @@
+    Report.result(Tests.create_terrain_spawner_entity, terrain_spawner_entity.id.IsValid())
+    height_provider_entity = hydra.Entity("HeightProviderEntity")
+    height_provider_entity.create_entity(azmath.Vector3(0.0, 0.0, 0.0), entity2_components_to_add,terrain_spawner_entity.id)
+    Report.result(Tests.create_height_provider_entity, height_provider_entity.id.IsValid())
+
+    # Give everything a chance to finish initializing.
+    general.idle_wait_frames(1)
+
+    # 4) Start the Tracer to catch any errors and warnings
+    with Tracer() as section_tracer:
+        # 5) Set the base Terrain World values
+        world_bounds_max = azmath.Vector3(1024.0, 1024.0, 1024.0)
+        world_bounds_min = azmath.Vector3(0.0, 0.0, 0.0)
+        height_query_resolution = azmath.Vector2(1.0, 1.0)
+        editor.EditorComponentAPIBus(bus.Broadcast, "SetComponentProperty", terrain_world_component, "Configuration|World Bounds (Max)", world_bounds_max)
+        editor.EditorComponentAPIBus(bus.Broadcast, "SetComponentProperty", terrain_world_component, "Configuration|World Bounds (Min)", world_bounds_min)
+        editor.EditorComponentAPIBus(bus.Broadcast, "SetComponentProperty", terrain_world_component, "Configuration|Height Query Resolution (m)", height_query_resolution)
+        world_max = hydra.get_component_property_value(terrain_world_component, "Configuration|World Bounds (Max)")
+        world_min = hydra.get_component_property_value(terrain_world_component, "Configuration|World Bounds (Min)")
+        world_query = hydra.get_component_property_value(terrain_world_component, "Configuration|Height Query Resolution (m)")
+        Report.result(Tests.bounds_max_changed, world_max == world_bounds_max)
+        Report.result(Tests.bounds_min_changed, world_min == world_bounds_min)
+        Report.result(Tests.height_query_changed, world_query == height_query_resolution)
+
+        # 6) Change the Axis Aligned Box Shape dimensions
+        box_dimensions = azmath.Vector3(SET_BOX_X_SIZE, SET_BOX_Y_SIZE, SET_BOX_Z_SIZE)
+        terrain_spawner_entity.get_set_test(0, "Axis Aligned Box Shape|Box Configuration|Dimensions", box_dimensions)
+        box_shape_dimensions = hydra.get_component_property_value(terrain_spawner_entity.components[0], "Axis Aligned Box Shape|Box Configuration|Dimensions")
+        Report.result(Tests.box_dimensions_changed, box_dimensions == box_shape_dimensions)
+        
+        # 7) Set the Vegetation Shape reference to terrain_spawner_entity
+        height_provider_entity.get_set_test(0, "Configuration|Shape Entity Id", terrain_spawner_entity.id)
+        entityId = hydra.get_component_property_value(height_provider_entity.components[0], "Configuration|Shape Entity Id")
+        Report.result(Tests.shape_changed, entityId == terrain_spawner_entity.id)
+
+        # 8) Set the FastNoise Gradient frequency to 0.01
+        Frequency = 0.01
```

Comment:
frequency or FREQUENCY. initial cap is for class or staticmethod if I recall.

Reply:
Changed